test/e2e/scenario/logs.scenario.ts

```diff
@@ -78,18 +78,15 @@ describe('logs', () => {
     .withLogs({ forwardErrorsToLogs: true })
     .run(async ({ serverEvents, baseUrl, servers }) => {
       await browserExecuteAsync((done) => {
-        fetch('/throw-large-response').then(
-          (response) => {
-            // The body stream needs to be cancelled, else the browser will still download the whole
-            // response even if it is unused.
-            response
-              .body!.getReader()
-              .cancel()
-              .catch((error) => console.log(error))
-            done(undefined)
-          },
-          (error) => console.log(error)
-        )
+        const controller = new AbortController()
+        const signal = controller.signal
+
+        setTimeout(() => {
+          controller.abort()
+          setTimeout(() => done(undefined), 1000)
+        }, 2000)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏those timeout usages seems strange to me, we should talk about it

Reply:
This wasn't ideal I agree. I was trying to call `controller.abort()` and `done()` on `.then` but found that no entry would be present in `serverEvents`.

This workaround gets around the issue. But I agree it isnt ideal